<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="currentUserLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current User:</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentUserRatingLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rating: 800 -&gt; 820</string>
   </property>
  </widget>
  <widget class="QLabel" name="dailyPuzzleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Daily Puzzle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="tournamentLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tournament</string>
   </property>
  </widget>
  <widget class="QLabel" name="matchupLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Niemann Vs Carlsen</string>
   </property>
  </widget>
  <widget class="QLabel" name="puzzleRatingLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rating: </string>
   </property>
  </widget>
  <widget class="QLabel" name="outcomeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Outcome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="userOutcomeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Win / Loss</string>
   </property>
  </widget>
  <widget class="QLabel" name="solutionLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Solution:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pgnSolutionLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>e4 e5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backMoveButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forwardMoveButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
